modules/nf-core/liftoff/main.nf

```diff
@@ -0,0 +1,62 @@
+process LIFTOFF {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/liftoff:1.6.3--pyhdfd78af_0':
+        'biocontainers/liftoff:1.6.3--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(target_fa)
+    path ref_fa, name: 'ref_assembly.fa'
+    path ref_annotation
```

Comment:
Will there ever be a situation where you would give the feature database `-db` instead of just the reference annotation? 

Maybe include it.